build/ci/evergreen.yml

```diff
@@ -580,7 +580,43 @@ tasks:
           MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud
-          E2E_TAGS: atlas,clusters
+          E2E_TAGS: atlas,clusters,flags
+  - name: atlas_clusters_file_e2e
```

Comment:
[q] any reason for the split of build variants?

Reply:
yes, I've seen some time being taken to deploy clusters on gov instances, this is splitting the time in 3. Also we had to add steps to wait for deletion to be able to delete projects so that is adding time to the tests.